plugins/content/lang/en/filter_content.php

```diff
@@ -0,0 +1,3 @@
+<?php
+
+$string['filtername'] = 'Content';
```

Comment:
Minor: Can we make this "OSX Content" or something like that? Only because I realized on the filter management page there's already use of `Content` in a different context and it confused me for a second :sweat_smile: